samoa-instances/src/main/java/org/apache/samoa/instances/Attribute.java

```diff
@@ -199,7 +196,14 @@ public String toString() {
     text.append(ARFF_ATTRIBUTE).append(" ").append(Utils.quote(this.name)).append(" ");
 
     if (isNominal) {
-      text.append(ARFF_ATTRIBUTE_NOMINAL);
+      text.append('{');
+      Enumeration enu =  enumerateValues();
+      while (enu.hasMoreElements()) {
+        text.append(Utils.quote((String) enu.nextElement()));
+        if (enu.hasMoreElements())
+          text.append(',');
+      }
+      text.append('}');
```

Comment:
Hi @fobeligi,
Was the previous one not working properly?
Do we need to construct the arff format explicitly?
Thanks!